crates/biome_js_analyze/src/lint/correctness/no_unused_private_class_members.rs

```diff
@@ -76,9 +80,27 @@ declare_node_union! {
     pub AnyMember = AnyJsClassMember | TsPropertyParameter
 }
 
+#[derive(Debug, Clone)]
+pub enum UnusedMemberAction {
+    Remove(AnyMember),
```

Comment:
nit: Maybe `RemoveMember(AnyMember)`? Calling it only `Remove` doesn't clarify how it differs from `RemovePrivateModifier`.